src/axom/mir/docs/sphinx/mir_clipping.rst

```diff
@@ -0,0 +1,137 @@
+|``fields:``                      | The fields node lets the caller provide a list of    |
+|                                 | field names that will be processed and added to the  |
+|                                 | output mesh. The form is *currentName:newName*. If   |
+|                                 | the *fields* node is not given, the algorithm will   |
+|                                 | process all input fields. If the fields node is empty|
+|                                 | then no fields will be processed.                    |
++---------------------------------+------------------------------------------------------+
+| ``inside: number``              | Indicates to the clipping algorithm that it should   |
+|                                 | preserve zone fragments that were "inside" the clip  |
+|                                 | boundary. Set to 1 to enable, 0 to disable. The      |
+|                                 | algorithm will generate these fragments by default.  |
++---------------------------------+------------------------------------------------------+
+| ``originalElementsField: name`` | The name of the field in which to store the original |
+|                                 | elements map.                                        |
++---------------------------------+------------------------------------------------------+
+| ``outside: number``             | Indicates to the clipping algorithm that it should   |
+|                                 | preserve zone fragments "outside" the clip boundary. |
+|                                 | Set to 1 to enable, 0 to disable. These fragments are|
+|                                 | not on by default.                                   |
++---------------------------------+------------------------------------------------------+
+| ``selectedZones: [zone list]``  | An optional argument that provides a list of zone ids|
+|                                 | on which to operate. The output mesh will only have  |
+|                                 | contributions from zone numbers in this list, if it  |
+|                                 | is given.                                            |
++---------------------------------+------------------------------------------------------+
+| ``topologyName: name``          | The name of the new topology in the output mesh. If  |
+|                                 | it is not provided, the output topology will have the|
+|                                 | same name as the input topology.                     |
++---------------------------------+------------------------------------------------------+
+
+##########
+ClipField
+##########
+
+To use the ``ClipField`` class, one must have Blueprint data with at least one vertex-associated
+field. Views for the coordset and topology are created and their types are used to instantiate
+a ``ClipField`` object. This takes a Conduit node for the input Blueprint mesh, a Conduit
```

Comment:
I'd start the sentence with "The `ClipField` constructor takes..." rather than "This takes..."

Reply:
Changed.